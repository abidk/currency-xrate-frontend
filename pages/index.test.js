import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";
import App from "./index.js";

describe(__filename, () => {
  it("rendered page should match snapshot", () => {
    const data = {
      rates: {
        base: { value: "EUR" },
        rates: [{ currency: { value: "GBP" }, quote: { value: "0.86075" } }],
      },
    };
    const component = renderer.create(<App data={data} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
