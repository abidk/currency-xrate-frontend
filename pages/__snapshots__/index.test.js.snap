// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Users/abid/Documents/currency-xrate/currency-xrate-frontend/pages/index.test.js rendered page should match snapshot 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="main"
        >
          <div
            className="exchange"
          >
            <h1
              className="mb-3"
            >
              Historical data
            </h1>
            <div
              className="card"
            >
              <div
                className="card-body"
              >
                <p>
                  Base Currency: 
                </p>
                <div
                  className="table-responsive-sm"
                >
                  <table
                    className="table"
                  >
                    <thead
                      className="thead-dark"
                    >
                      <tr>
                        <th
                          scope="col"
                        >
                          Currency
                        </th>
                        <th
                          scope="col"
                        >
                          Value
                        </th>
                      </tr>
                    </thead>
                    <tbody>
                      <tr>
                        <td>
                          GBP
                        </td>
                        <td>
                          0.86075
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
